### PR TITLE
Add 'recording company' lemma

### LINKED ISSUE #176: Add 'recording company'
As new member for 'record company'

### PR DESCRIPTION
**Pull request by jmccrae**
Fix #176

### CODE REVIEW

**Comment by arademaker** [comment]
I see the addition of the new lemma,  but not the change of the synset associating this new lemma  to the synset.

**Comment by jmccrae** [comment]
New lemmas don't change the synset

File: src/wn-noun.group.xml [hunk 1]
```diff
@@ -17722,6 +17722,10 @@
       <Lemma writtenForm="Nancahuazu Guerrilla" partOfSpeech="n"/>
       <Sense id="ewn-Nancahuazu_Guerrilla-n-08051986-00" n="0" synset="ewn-08051986-n"/>
     </LexicalEntry>
+    <LexicalEntry id="ewn-recording_company-n">
+      <Lemma writtenForm="recording company" partOfSpeech="n"/>
+      <Sense id="ewn-recording_company-n-08495272-02" n="0" synset="ewn-08495272-n"/>
```
Comment (arademaker) [inline src/wn-noun.group.xml]: this is the line that links this entry as a new lexical entry in the synset?

Comment (jmccrae) [reply src/wn-noun.group.xml]: Yes, indeed.

Comment (arademaker) [reply src/wn-noun.group.xml]: ok. thanks. now clear.

**Pull request merged**